gce/worker/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN chmod +x /usr/local/bin/dumb-init
 RUN conda install --yes -c conda-forge \
     cytoolz \
     dask=0.17.0 \
-    distributed=1.21.0 \
+    distributed \
```

Comment:
should we pin this version number to the same as the notebook dockerfile?

Reply:
Sure?  It shouldn't matter that much both because of the pip install just below and because I expect to overwrite this once 1.21.3 gets into defaults.

Reply:
Generally speaking though I agree that we should probably stick to a hygiene of pinning things when possible

Reply:
Removed entirely